src/context.ts

```diff
@@ -480,8 +480,8 @@ export class Context<
 ```
    */
   set: Set<E> = (key: string, value: unknown) => {
-    this._var ??= {}
-    this._var[key as string] = value
+    // @ts-expect-error this._var is initialized as {}
```

Comment:
It throws a type error that points to the possibility of `undefined`. But it has been initialized above. So ignore it.